examples/svg_render/src/main.rs

```diff
@@ -240,10 +264,10 @@ fn update_inputs(scene: &mut Scene, event_loop: &mut glutin::EventsLoop) -> bool
                 VirtualKeyCode::Escape => {
                     status = false;
                 }
-                VirtualKeyCode::PageDown => {
```

Comment:
I'd prefer to keep PgUp/down to control the zoom like in the other examples and the cli app.